src/akkudoktoreosserver/flask_server.py

```diff
@@ -132,9 +140,13 @@ def flask_gesamtlast():
 @app.route("/gesamtlast_simple", methods=["GET"])
 def flask_gesamtlast_simple():
     if request.method == "GET":
-        year_energy = float(
-            request.args.get("year_energy")
+        year_energy = request.args.get(
+            "year_energy"
         )  # Get annual energy value from query parameters
```

Comment:
if it is the year, then the following comment will be more precise:
"get year for predicted annual energy demand".